### PR TITLE
Add 'large_attr' data structure in op definition for large chunk (data of pandas/numpy)

### PR DESCRIPTION
**Pull request by lidongze0629**
## What do these changes do?
| size      | time(s) |
| ----------- | ----------- |
| 934M (vertices)      | 42.56s       |

## Related issue number

links #1290

### CODE REVIEW

File: analytical_engine/core/grape_instance.cc [hunk 1]
```diff
@@ -1058,7 +1058,7 @@ bl::result<void> GrapeInstance::registerGraphType(const rpc::GSParams& params) {
 bl::result<std::shared_ptr<DispatchResult>> GrapeInstance::OnReceive(
     const CommandDetail& cmd) {
   auto r = std::make_shared<DispatchResult>(comm_spec_.worker_id());
-  rpc::GSParams params(cmd.params);
+  rpc::GSParams params(cmd.params, cmd.large_attr);
```
Comment (sighingnow) [inline analytical_engine/core/grape_instance.cc:1061]: Will the `cmd.large_attr` be copied? or just a reference?

**Review by sighingnow — APPROVED** [review]

File: analytical_engine/core/grape_instance.cc (hunk 1 above)
Comment (lidongze0629) [reply analytical_engine/core/grape_instance.cc:1061]: I will double-check any possible copy both in c++ and python side.

File: analytical_engine/core/io/property_parser.h [hunk 2]
```diff
@@ -134,46 +132,43 @@ struct Graph {
...
+  vertex->values = data;
+  graph->vertices.push_back(vertex);
 }
 
-inline detail::Edge::SubLabel ParseSubLabel(const AttrMap& attrs) {
+inline void ParseEdge(std::shared_ptr<detail::Graph>& graph,
+                      const std::string& data, const AttrMap& attrs) {
+  std::string label = attrs.at(rpc::LABEL).s();
+
+  bool label_edge_exists = false;
```
Comment (siyuan0322) [inline analytical_engine/core/io/property_parser.h:149]: rename to `has_edge_label`?

File: analytical_engine/core/server/command_detail.h [hunk 3]
```diff
@@ -38,17 +38,23 @@ struct CommandDetail {
   CommandDetail() = default;
 
   CommandDetail(const rpc::OperationType& op_type,
-                std::map<int, rpc::AttrValue>&& op_params)
-      : type(op_type), params(std::move(op_params)) {}
+                std::map<int, rpc::AttrValue>&& op_params,
+                rpc::LargeAttrValue large_attr)
```
Comment (siyuan0322) [inline analytical_engine/core/server/command_detail.h:42]: use `rpc::LargeAttrValue&&` ?

File: analytical_engine/core/server/command_detail.h [hunk 4]
```diff
@@ -38,17 +38,23 @@ struct CommandDetail {
...
+      : type(op_type),
+        params(std::move(op_params)),
+        large_attr(std::move(large_attr)) {}
 
   CommandDetail(const rpc::OperationType& op_type,
-                std::map<int, rpc::AttrValue>&& op_params, rpc::QueryArgs args)
+                std::map<int, rpc::AttrValue>&& op_params,
+                rpc::LargeAttrValue large_attr, rpc::QueryArgs args)
```
Comment (siyuan0322) [inline analytical_engine/core/server/command_detail.h:49]: ditto

File: analytical_engine/core/server/rpc_utils.h [hunk 5]
```diff
@@ -94,8 +94,9 @@ inline rpc::ReportType get_param_impl<rpc::ReportType>(
  */
 class GSParams {
  public:
-  explicit GSParams(std::map<int, rpc::AttrValue> params)
-      : params_(std::move(params)) {}
+  explicit GSParams(std::map<int, rpc::AttrValue> params,
+                    const rpc::LargeAttrValue& large_attr)
```
Comment (siyuan0322) [inline analytical_engine/core/server/rpc_utils.h:98]: you can't move a const reference. use &&

**Review by siyuan0322 — APPROVED** [review]

**Pull request merged**